stable/database/templates/statefulset.yaml

```diff
@@ -313,7 +313,7 @@ spec:
       initContainers:
       - name: init-disk
         image: {{ template "init.image" . }}
-        imagePullPolicy: {{ default "" .Values.busybox.imagepullPolicy | quote }}
+        imagePullPolicy: {{ default "" .Values.busyboximagepullPolicy | quote }}
```

Comment:
I don't this line is correct.

The values in the values file are `nuodb.image.pullPolicy` and `busybox.image.pullPolicy`
There is also a template in `_helpers.yaml` called `nuodb.imagePullSecrets`

This new line seems to be a confusion of the 2.

I presume the original error is the missing of the period between `image` and `pullPolicy`.

So perhaps: `.Values.busybox.imagepullPolicy`
should be: `.Values.busybox.image.pullPolicy`